magpie/ui/management/templates/add_group.mako

```diff
@@ -71,6 +71,31 @@
             <td>
             </td>
         </tr>
+        <tr>
+            <td>Terms and conditions:</td>
+            <td>
+                <label>
+                    <input type="text" name="terms" value="${form_terms}" />
```

Comment:
Hard to know with static analysis. 
Is this field sufficiently large for convivial writing of long blocks of T&C text? 
I think the field is by default not expandable and single line. 

Might need to increase its size for this case + ensure text wrapping is used to help writing paragraphs instead of horizontally scrolling text box.
You can make a specific CSS class to apply needed styles to apply this behaviour if other similar cases arise.

Reply:
Good idea, I didn't think to test for long blocks of text, I will look into it!

Reply:
I am currently checking to make it better for multiline text. I use a `textarea`, that can be resized only vertically. By default, it displays 4 rows, but can be resized to display more, and can be scrolled through too. What do you think of the layout? 

This is just a fast draft, but I think I should make the `Terms and conditions` and its associated `(optional)` label aligned with the top of the `textarea`. 

Also, I should align the `(optional)` label of the `description` beside its input text box, and not aligned with the `textarea` of the T&C. 

There is the `Add Group` button that should stay aligned as before, with the input text boxes of `Group name` and `Description`, or it will move depending of the horizontal size of the `textarea`. You can see it has a weird alignment on the current screenshot.

Also, what do you think of the horizontal size of the `textarea`? I didn't want to make the same size as the others, as it should contain more text and should be larger. I think I should leave the horizontal resizing enabled too, so the user can use the width he wants.

Let me know if my idea makes sense or if you have any better suggestion so I can work on a proper version of the ui!

![image](https://user-images.githubusercontent.com/36516122/134383502-b5da7a85-73d8-48c6-be0b-e6ae6a25645d.png)

Reply:
I've added a minimal width and height, to avoid weird formatting, and realigned the different elements.

Here is how it looks, by default, when opening the page :
![image](https://user-images.githubusercontent.com/36516122/134398683-26cf1469-ad87-4127-bdaa-6b3a3240a3d0.png)

Here is an example with the minimal size (the default width is a bit larger) :
![image](https://user-images.githubusercontent.com/36516122/134398368-8574a2f1-ca48-4ba5-9412-f4e33b284c92.png)

Here is an example with the expanded textarea (the different elements mentioned before are now aligned):
![image](https://user-images.githubusercontent.com/36516122/134398423-f7523f4d-d0fa-4597-bc52-25374a40089e.png)

Reply:
> I use a textarea, that can be resized only vertically. By default, it displays 4 rows, but can be resized to display more, and can be scrolled through too. What do you think of the layout?

Excellent!

> This is just a fast draft, but I think I should make the Terms and conditions and its associated (optional) label aligned with the top of the textarea.

I totally agree.

> Also, I should align the (optional) label of the description beside its input text box, and not aligned with the textarea of the T&C.
> Also, what do you think of the horizontal size of the textarea? I didn't want to make the same size as the others, as it should contain more text and should be larger. I think I should leave the horizontal resizing enabled too, so the user can use the width he wants.

The last screenshot looks perfect. 
If `(optional)` works by automatically following resizes of T&C, let's keep that design.